var/spack/repos/builtin/packages/sgpp/package.py

```diff
@@ -49,8 +49,8 @@ class Sgpp(SConsPackage):
     # Continue despite distutils deprecation warning!
     # distutils will be removed in future SGpp versions. See
     # https://github.com/SGpp/SGpp/issues/263 for associated issue!
-    # TODO Once distutils is removed from SGpp, limit patch to @:3.4.0
-    patch("disable_disutils_deprecation_warning.patch", when="^python@3.10:3.11")
+    patch("disable_disutils_deprecation_warning.patch", when="@:3.4.0 ^python@3.10:3.11")
+    patch("for_aarch64.patch", when="target=aarch64:")
```

Comment:
Can you add a brief comment above this `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
I have updated the description of the disable_disutils_deprecation_warning.patch and linked the relevant issue within sgpp.

For the aarch64.patch: This one is only changed by the PR since I moved it up a few lines in the package.py recipe (as I found it odd to not have it together with the other patches within the package). It was added quite a while ago to the spack package in #18968. Of course, I still added a brief description for it.